### ISSUE TITLE
Unable to scrape metrics with ClusterPodMonitoring "monitoring.googleapis.com/v1"

### ISSUE THREAD

**Issue by marandalucas** [mongodb/mongodb-kubernetes-operator#1217]
**What did you do to encounter the bug?**

**We can not scrape Prometheus metrics in GCP using Managed Prometheus.**

We are trying to scrape metrics from MongoDB following the next example:
https://github.com/mongodb/mongodb-kubernetes-operator/blob/93c86675b851e7d44d5bfbd65375602dd4b3ad49/docs/prometheus/mongodb-prometheus-sample.yaml#L75-L83

**Scenario:**
We are using Managed Prometheus in GCP with GKE. Because of that, we have to [convert](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#gmp-servicemonitor) _ServiceMonitor_ to _ClusterPodMonitoring_ or _PodMonitoring_.

In this scenario, we can not use "_authentication_", because unfortunately is not allowed -> 
https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.5.0/doc/api.md#scrapeendpoint

- **Could you help us? Did anyone have the same problem??**
- **Cloud you add a feature to scrape metrics without auth?**

Thank you so much

_ClusterPodMonitoring Example:_
```
---
apiVersion: monitoring.googleapis.com/v1
kind: ClusterPodMonitoring
metadata:
  name: mongodb-cluster
  namespace: gmp-system
  labels:
    name: mongodb-cluster
spec:
  selector:
    matchLabels:
      app: mongodb-rs-srv-svc
  endpoints:
  - port: prometheus
    interval: 15s
  # - authentication:
  #     password:
  #       key: password
  #       name: mongodb-rs-srv-metrics-endpoint-password
  #     username:
  #       key: username
  #       name: mongodb-rs-srv-metrics-endpoint-password
```

**What did you expect?**
Scrape metrics without basic auth or another workaround to solve it.

**Comment by marandalucas** [comment]
Hey! Is there good news? We also opened a ticket to GCP to review the case. 

We don't know why... Managed Prometheus does not allow to convert ServiceMonitor to ClusterPodMonitoring with - authentication: 

Thanks in advance

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
